ote/src/cljs/ote/app/controller/route/route_wizard.cljs

```diff
@@ -251,6 +251,10 @@
        (every? valid-calendar-rule?
                (::transit/service-rules calendar))))
 
+(defn- interval< [a b]
+  (when (and a b)
+    (< (time/minutes-from-midnight a) (time/minutes-from-midnight b))))
```

Comment:
Does this work if interval has years and months?

Reply:
Was not intended to, on current UI it's possible to enter only HH:mm.

Reply:
Thanks, improvements pushed to branch.